spec/features/profiles_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+feature "zarządzanie profilem użytkownika" do
+  context "po zalogowaniu" do
+    context "profil użykownika w bazie" do
+      before do
+        Person.create!(name: "Anna", surname: "Genialna", email: "a.genialna@gmail.com",
+                      sex: "kobieta")
+      end
+      scenario "sprawdzanie możliwości edycji profilu" do
+        visit '/profile'
+      
+        expect(page).to have_css('a[title="Edytuj swoje dane"]')
+        expect(page).to have_css('a[title="Edytuj hasło"]')
+      end
+    
+      scenario "edytowanie profilu użytkownika" do
```

Comment:
Tutaj też wcześniej trzeba się zalogować jako A. Genialna. Można to zrobić w bloku `before` tak żeby nie pisać tego kilka razy.